integration-tests/test-suite/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "fluent-bit-test-suite",
+  "version": "1.0.0",
+  "description": "Validates fluent bit end-to-end with several configurations",
+  "scripts": {
+    "test": "jest --verbose fluent-bit-tests.test.js"
+  },
+  "author": "logging-team@newrelic.com",
+  "license": "ISC",
+  "devDependencies": {
+    "axios": "^1.3.2",
+    "axios-retry": "^3.4.0",
+    "eslint": "^8.33.0",
+    "eslint-plugin-jest": "^27.2.1",
+    "eslint-plugin-prettier": "^4.2.1",
+    "jest": "^29.4.2",
+    "jest-html-reporter": "^3.7.0",
+    "jest-junit": "^15.0.0",
+    "prettier": "^2.8.4",
+    "winston": "^3.8.2"
+  },
+  "jest-junit": {
+    "outputDirectory": "reports/tests",
+    "outputName": "test-report.xml"
+  },
+  "jest-html-reporter": {
+    "outputPath": "./reports/tests/test-report.html",
+    "includeConsoleLog": true,
+    "includeFailureMsg": true,
+    "includeSuiteFailure": true
+  }
+}
```

Comment:
Ask in #help-opensource whether we need to include any license notes (dependency manifest) in the project.